docs/gamecube.md

```diff
@@ -0,0 +1,48 @@
+---
+section: legacy
+hide:
+  - next
+  - prev
+  - search
+  - tabs
+---
+
+##Gamecube Controller Setup with Official WiiU Adapter
+
+Optional video walkthrough: <https://www.youtube.com/watch?v=4ntXPY4SqNQ>
```

Comment:
I'd prefer not to have raw URLs if we can help it. But not sure how to do this in a nice way? Maybe just a red button (or normal button with the red YouTube logo, that says `Optional video Walkthrough`? Position is fine.

`[:fontawesome-brands-youtube: Optional Video Walkthrough](https://www.youtube.com/watch?v=4ntXPY4SqNQ){ .md-button data-md-color-primary="red" data-md-color-accent="red" }`

![image](https://user-images.githubusercontent.com/1178816/211361933-0ce3ab19-6617-4a90-8a86-8387bf11e320.png)


The other links in the guide are easier, I put a comment for each of those.

Reply:
Fixed in https://github.com/smash64-dev/smash64-online/pull/10/commits/3168e0193f3172017cdee399d0f3b9137c64d788. Button looks good, had no idea you could do that.